Service/EnvironmentService.php

```diff
@@ -63,6 +154,27 @@ public function getEnvironmentNames(): array
         return array_keys($this->getEnvironments());
     }
 
+    public function getNotSnapshotEnvironments(): array
```

Comment:
Ok because POC has priority, but should be refactored using the strategy from #281